src/FunctionApps/SilverTransforms/Transform/geo.py

```diff
@@ -0,0 +1,68 @@
+import re
+
+from typing import TypedDict
+
+from pymongo.collection import Collection
+from smartystreets_python_sdk import StaticCredentials, ClientBuilder
+from smartystreets_python_sdk import us_street
+from smartystreets_python_sdk.us_street.lookup import Lookup
+
+
+class GeocodeResult(TypedDict):
+    key: str
+    address: str
+    zipcode: str
+    fips: str
+    lat: float
+    lng: float
+
+
+def cached_geocode(
```

Comment:
Not blocking PR Merge, but after talking to other CDC folks, I had mentioned the notion of "caching" results because it would make things more efficient. For example, with Lab test data, you will often get repeated patients who test several times, so once you've seen them and confirm they are still listing the same address, you don't have to "re-geocode". There's some issues here in terms of compliance in the sense of whether we're allowed to store patient data over a period of time. But in this case you're not really storing patient data, you're just storing the address and geocoded results themselves. I wonder how that shakes out in terms of a compliance issue?

Reply:
definitely agreed - for that matter though, we really probably need to clear smartystreets more generally. Happy to disable caching until we figure it out if that makes sense